### DISCUSSION TITLE
Esp32s2 using OTG + webserver cam

### DISCUSSION THREAD

**Discussion by atamico** [hathach/tinyusb#1319] [Q&A] [answered]
I would like to know if anyone has already managed/tried to configure the ESP32s2 on communication pins 19, 20 as a USB-OTG port to connect other devices.

I want to make a Webserver with local ip, the same as ESP-CAM does, however, I want to use a USB Endoscopic USB camera (which is not possible with ESP-CAM).

I need help on How to use ESP32s2:
- Creating the Webserver_Cam
- Using USB model camera
- For Local ip.

I know it's possible because I saw it in an advertisement for "espressif".
~> https://youtu.be/RQOhrgRd7N8?t=55

But it looks like the code that does the communication is TinyUsb 

Can someone help me?

**Comment by me-no-dev** [comment, marked as answer]
This question is not for this repository. Try here: https://github.com/espressif/esp-idf

**Reply by atamico** [reply]
ok, tanks
